csharp/Platform.Collections/Arrays/ArrayPool[T].cs

```diff
@@ -21,15 +19,43 @@ public class ArrayPool<T>
         private readonly int _maxArraysPerSize;
         private readonly Dictionary<long, Stack<T[]>> _pool = new Dictionary<long, Stack<T[]>>(ArrayPool.DefaultSizesAmount);
 
+        /// <summary>
+        /// <para>Initializes a new instance of the ArrayPool class.</para>
+        /// <para>Инициализирует новый экземпляр класса ArrayPool.</para>
+        /// </summary>
+        /// <param name="maxArraysPerSize"><para>The maximum size of the array in the pool.</para><para>Максимальный размер массива в пуле.</para></param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public ArrayPool(int maxArraysPerSize) => _maxArraysPerSize = maxArraysPerSize;
 
+        /// <summary>
+        /// <para>Initializes a new instance of the ArrayPool class with a default argument.</para>
+        /// <para>Инициализирует новый экземпляр класса ArrayPool с аргументом по умолчанию.</para>
+        /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public ArrayPool() : this(ArrayPool.DefaultMaxArraysPerSize) { }
 
+        /// <summary>
+        /// <para>Retrieves an array of the specified size from the array pool, which is automatically freed after calling the Disposable destructor.</para>
+        /// <para>Извлекает из пула массивов массив с указанным размером, который автоматически освободится, после вызова Disposable деструктора.</para>
+        /// </summary>
+        /// <param name="size"><para>The allocated array size.</para><para>Размер выделяемого массива.</para></param>
+        /// <returns>
+        /// <para>An array of the specified size from the array pool, if it is not in the pool, a new one is created.</para>
+        /// <para>Массив указанного размера из пула массивов, если его нет в пуле, cоздаётся новый.</para>
+        /// </returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public Disposable<T[]> AllocateDisposable(long size) => (Allocate(size), Free);
 
+        /// <summary>
+        /// <para>Changes the number of elements in the array to the specified value.</para>
+        /// <para>Изменяет количество элементов массива до указанной величины.</para>
+        /// </summary>
+        /// <param name="source"><para>Source array.</para><para>Исходный массив.</para></param>
+        /// <param name="size"><para>The size of the new array.</para><para>Размер нового массива.</para></param>
```

Comment:
The array can be the same if the size is the same. Not `new array`, but `new size`.